### DISCUSSION TITLE
Sync Whatsapp Templates via Twillio

### DISCUSSION THREAD

**Discussion by SwimGeek** [chatwoot/chatwoot#10240] [Self hosted]
Hi

Any advice for syncing approved (non media) Whatsapp templates via Twillio?

Templates were approved a few hours ago, but they don't seem to show up in self-hosted Chatwoot.

I'm wondering if template sync is supported when using Twillio?

Running Chatwoot via Docker. The INFO log level is crazy busy. How do I figure out why templates are not syncing?

Can't see to find docs about Whatsapp template features.

**Comment by sojan-official** [comment]
The whatsapp template selector in Chatwoot only supports the cloud api channel . it doesn't work with twilio at the moment.
